app/views/admin/CodeLogsView.coffee

```diff
@@ -0,0 +1,46 @@
+RootView = require 'views/core/RootView'
+template = require 'templates/admin/codelogs'
+CocoCollection = require 'collections/CocoCollection'
+CodeLog = require 'models/CodeLog'
+utils = require 'core/utils'
+
+
+module.exports = class CodeLogsView extends RootView
+  template: template
+  id: 'code-logs-view'
+  events:
+    'click .playback': 'onClickPlayback'
+    'blur #tooltip': 'onBlurTooltip'
+  constructor: (options) ->
+    @spade = new Spade()
+    super options
+    @codelogs = new CocoCollection([],
+      url: '/db/codelogs'
+      model: CodeLog
+    )
+    @listenTo(@codelogs, 'sync', @onThangsLoaded)
+    @supermodel.loadCollection(@codelogs, 'codelogs')
+      
+  onThangsLoaded: ->
+    @renderSelectors '#code-log-table'
+
+  onClickPlayback: (e) ->
+    @deleteTooltip()
+    events = LZString.decompressFromUTF16($(e.target).data('codelog'))
+    events = @spade.expand(JSON.parse(events))
+    tooltip = document.createElement('textarea')
+    tooltip.style.position = 'absolute'
+    tooltip.style.left = e.pageX + 20 + "px"
+    tooltip.style.top = e.pageY + "px"
+    tooltip.style.width = "512px"
+    tooltip.style.height = "512px"
+    tooltip.style.borderRadius = "8px"
+    tooltip.id = "codelogs-tooltip"
+    tooltip.addEventListener 'blur', @onBlurTooltip
+    document.body.appendChild tooltip
```

Comment:
We usually use jQuery to do DOM manipulation and event listeners like this. Not saying you have to, but for consistency, maybe better? Also I don't think the memory management here is enough; maybe if you add a `destroy` method (like most views have) that will get rid of any remaining tooltips. And do we need to remove the event listener ourselves or not when removing it from the DOM?

We should probably have this tooltip have a CSS class that we can style in our normal stylesheet, independent of the positioning bit (which could stay here).